src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx

```diff
@@ -294,6 +294,19 @@ export default class ComparisonGroupManager extends React.Component<IComparisonG
                                 placeholder="Enter a name for your new group"
                                 value={this.inputGroupName}
                                 onChange={this.onChangeInputGroupName}
+                                onKeyPress={
+                                    async(event) => {
+                                        if(event.key == 'Enter' && selectedSamples && this.inputGroupName.length && allGroupNames && (allGroupNames.indexOf(this.inputGroupName) === -1)){
+                                            await comparisonClient.addGroup(getGroupParameters(
```

Comment:
lets extract and resuse this logic since it's same as in click handler for button.

Reply:
@alisman Thanks! Updated.